controllers/cloud.redhat.com/providers/iqe/impl.go

```diff
@@ -53,20 +54,13 @@ func CreateIqeJobResource(cache *providers.ObjectCache, cji *crd.ClowdJobInvocat
 	} else {
 		tag = app.Spec.Testing.IqePlugin
 	}
-	plugin := app.Spec.Testing.IqePlugin
 	iqeImage := env.Spec.Providers.Testing.Iqe.ImageBase
 
-	constructedIqeCommand, err := ConstructIqeCommand(cji, plugin)
-	if err != nil {
-		return err
-	}
-
 	j.Spec.Template.Spec.ServiceAccountName = fmt.Sprintf("iqe-%s", app.Spec.EnvName)
 
 	c := core.Container{
 		Name:         nn.Name,
 		Image:        fmt.Sprintf("%s:%s", iqeImage, tag),
-		Command:      constructedIqeCommand,
```

Comment:
And `command `is removed in favor of the entry point script in the container: https://gitlab.cee.redhat.com/insights-qe/iqe-dockerfiles/-/merge_requests/59/diffs#bf13662e10e3e9768d5f98852899f5970db729dd_4_3